README.rst

```diff
@@ -141,7 +136,55 @@ How to use this tool
       E.g.:
       box build with local iso: iosxr-xrv64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso
       box build with remote iso: iosxr-xrv64-vbox/iosxr_iso2vbox.py user@server:/myboxes/iosxrv-fullk9-x64.iso
-      box build with ova export, verbose and upload to artifactory: iosxr-xrv64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso -o -v -a 'New Image'
+      box build with ova export and verbose: iosxr-xrv64-vbox/iosxr_iso2vbox.py iosxrv-fullk9-x64.iso -o -v
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The iosxr_store.py tool is:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A tool written to copy a generated box to a repository, with a
+generated message to an alias.
+
+^^^^^^^^^^^^^^^^^^^^^^
+How to use this tool
+^^^^^^^^^^^^^^^^^^^^^^
+
+::
+
+   rwellum@RWELLUM-M-34DF:[~/Desktop/Boxes]: iosxrv-x64-vbox/iosxr_store_box.py -h
```

Comment:
I'd suggest removing your prompt string and replacing it with a generic prompt like `$`:

`$ iosxrv-x64-vbox/iosxr_store_box.py -h`